test/unit_tests/arkode/CXX_serial/CMakeLists.txt

```diff
@@ -50,10 +50,17 @@ foreach(test_tuple ${unit_tests})
       ${CMAKE_SOURCE_DIR}/include
       ${CMAKE_SOURCE_DIR}/src)
 
-    # Libraries to link against
+    # We explicitly choose which object libraries to link to
+    # and link in the arkode objects so that we have access to
+    # private functions w/o changing their visibility in the installed libraries.
```

Comment:
Nice! Are similar changes need to other unit tests to resolve #139? I think the ARKODE tests might be the only ones that access internal functions currently.

Reply:
It should only be needed for the tests that use internal functions.

Reply:
It seems my fix may not work on all systems or all versions of CMake that we support though, so I may need to revert this and turn off the unit tests on Windows.